node/service/src/service.rs

```diff
@@ -461,23 +462,21 @@ where
 		_,
 		_,
 		_,
-		_,
 	>(cumulus_client_consensus_aura::ImportQueueParams {
 		block_import: client.clone(),
 		client: client.clone(),
 		create_inherent_data_providers: move |_, _| async move {
-			let time = sp_timestamp::InherentDataProvider::from_system_time();
+			let timestamp = sp_timestamp::InherentDataProvider::from_system_time();
 
 			let slot =
 				sp_consensus_aura::inherents::InherentDataProvider::from_timestamp_and_slot_duration(
-					*time,
+					*timestamp,
 					slot_duration,
 				);
 
-			Ok((time, slot))
+			Ok((slot, timestamp))
```

Comment:
These order swaps worry me. It feels like a mistake.

Reply:
Swaps happen due to changes in cumulus, i think these are initial sign of parting collator from relay 
https://github.com/substrate-developer-hub/substrate-parachain-template/pull/133